CONTRIBUTING.md

```diff
@@ -75,6 +75,24 @@ flake8 `git diff main --name-only`
 ufmt format `git diff main --name-only`
 ```
 
+### Type checking
+
+TorchMultimodal uses mypy for type checking. You can install mypy with the command
+
+```
+python3 -m pip install mypy
```

Comment:
can u just do pip install like others?

Reply:
Maybe, but this is from the [mypy docs](https://mypy.readthedocs.io/en/stable/getting_started.html), so personally I would stick with the officially-supported command